frontend/public/components/factory/list-page.tsx

```diff
@@ -210,7 +268,7 @@ export const FireMan_ = connect(null, { filterList })(
       if (canCreate) {
         if (createProps.to) {
           createLink = (
-            <Link className="co-m-primary-action" {...createProps}>
+            <Link className="co-m-primary-action" to={createProps.to}>
```

Comment:
Are there any other createProps we might need to still pass like `onClick` or `id`?

Reply:
we could, but looking at the current usages, I didnt find any which would pass anything else beside `to` prop